platforms/gce.env

```diff
@@ -0,0 +1,7 @@
+# GCE E2E ENVS
```

Comment:
eh, I feel like I can get rid of it by passing all args through `config.json`.. if it's not optimal I can convert it back to `foo.env`